common/on_actions/00_on_actions.txt

```diff
@@ -14,6 +14,7 @@ on_startup = {
 		WCGOV.7980				# Election timer
 		WCSUP.40				# Gives infertile trait
 		WCSUP.50				# Debuger: shows children with adult portraits
+		WCUTF.1				#Closes down Ulduar
```

Comment:
Why do you make event to set modifier if you can set modifier in province history?